builder/source/simple.go

```diff
@@ -112,50 +113,52 @@ func (s *SimpleSource) IsFetched() bool {
 	return PathExists(s.GetPath(s.validator))
 }
 
-// download utilises CURL to do all downloads
+// download simple files using go grab
```

Comment:
Go's convention is to use third-person sentences. That would be "download downloads simple files using go grab".
I'd avoid documenting internals, but that's just me.